### ISSUE TITLE
outputDirectory default of null invalid

### ISSUE THREAD

**Issue by milesfrain** [nwolverson/vscode-ide-purescript#145]
Should the default be `""` instead of `null`?

```
"purescript.outputDirectory": {                    
    "type": "string",                    
    "default": null,                    
    "description": "Override purs ide output directory (output/ if not specified
    "scope": "resource"                    
}, 
```

![image](https://user-images.githubusercontent.com/3578681/82393919-e6be4f80-9a04-11ea-8a46-f0fcbea6586d.png)

**Comment by nwolverson** [comment]
I think the type should ideally not have to use an empty string when blank, oddly this worked in the past - and I don't see any warning in my VSCode on this machine - requires a LS change to treat blank as null currently is treated.

**Comment by milesfrain** [comment]
Fixed with vscode update to 1.47.2
